slow_query: update the doc (#9036)
pingcap/docs: identify-slow-queries.md

* `Query_time`: The execution time of a statement.
* `Parse_time`: The parsing time for the statement.
* `Compile_time`: The duration of the query optimization.
* `Optimize_time`: The optimization time for the query plan.
* `Wait_TS`: The waiting time of the transaction to get timestamps.
* `Query`: A SQL statement. `Query` is not printed in the slow log, but the corresponding field is called `Query` after the slow log is mapped to the memory table.
* `Digest`: The fingerprint of the SQL statement.
* `Txn_start_ts`: The start timestamp and the unique ID of a transaction. You can use this value to search for the transaction-related logs.
* `Is_internal`: Whether a SQL statement is TiDB internal. `true` indicates that a SQL statement is executed internally in TiDB and `false` indicates that a SQL statement is executed by the user.
* `Index_names`: The index names used by the statement.

> * `Stats`: The health state of tables. `pseudo` indicates that the statistics is unhealthy.

Reviewer: ```suggestion
* `Stats`: The health state of the involved tables. `pseudo` indicates that the statistics are unhealthy.
```

* `Index_ids`: The IDs of the indexes involved in a statement.
* `Succ`: Whether a statement is executed successfully.
* `Backoff_time`: The waiting time before retry when a statement encounters errors that require a retry. The common errors as such include: `lock occurs`, `Region split`, and `tikv server is busy`.
* `Plan`: The execution plan of the statement. Use the `select tidb_decode_plan('xxx...')` statement to parse the specific execution plan.
* `Prepared`: Whether this statement is a `Prepare` or `Execute` request or not.
* `Plan_from_cache`: Whether this statement hits the execution plan cache.

> * `Plan_from_binding`: Whether this statement has bound execution plans.

Reviewer 2: ```suggestion
* `Plan_from_binding`: Whether this statement uses the bound execution plans.
```

* `Has_more_results`: Whether this statement has more results to be fetched by users.
* `Rewrite_time`: The time consumed for rewriting the query of this statement.
* `Preproc_subqueries`: The number of subqueries (in the statement) that are executed in advance. For example, the `where id in (select if from t)` subquery might be executed in advance.
* `Preproc_subqueries_time`: The time consumed for executing the subquery of this statement in advance.
* `Exec_retry_count`: The retry times of this statement. This field is usually for pessimistic transactions in which the statement is retried when the lock is failed.
* `Exec_retry_time`: The execution retry duration of this statement. For example, if a statement has been executed three times in total (failed for the first two times), `Exec_retry_time` means the total duration of the first two executions. The duration of the last execution is `Query_time` minus `Exec_retry_time`.

> * `KV_total`: The time consumed for all the RPC requests on TiKV/TiFlash.

Reviewer: ```suggestion
* `KV_total`: The time consumed on all the RPC requests on TiKV/TiFlash.
```

> * `PD_total`: The time consumed for all the RPC requests on PD.

Reviewer: consumed on

> * `PD_total`: The time consumed for all the RPC requests on PD.

Author: ```suggestion
* `PD_total`: The time spent on all the RPC requests on PD.
```

> * `Backoff_total`: The time consumed for all the backoff in the execution of this statement.

Author: ```suggestion
* `Backoff_total`: The time spent on all the backoff in the execution of this statement.
```

> * `Write_sql_response_total`: The time consumed for sending results to the client.

Reviewer: time spent on

> * `Write_sql_response_total`: The time consumed for sending results to the client.

Author: ```suggestion
* `Write_sql_response_total`: The time spent on sending results to the client.
```

* `Result_rows`: The row count of the result.

> * `IsExplicitTxn`: Whether this statement is in an explicit transaction. If the value is `false`, the transaction is `autocommit=1` and the statement is committed after execution.

Reviewer: ```suggestion
* `IsExplicitTxn`: Whether this statement is in an explicit transaction. If the value is `false`, the transaction is `autocommit=1` and the statement is automatically committed after execution.
```

* `Query_time`: The execution time of a statement.
* `Parse_time`: The parsing time for the statement.
* `Compile_time`: The duration of the query optimization.
* `Optimize_time`: The time consumed for optimizing the execution plan.

> * `Wait_TS`: The waiting time of the transaction to get timestamps.

Reviewer: ```suggestion
* `Wait_TS`: The waiting time of the statement to get transaction timestamps.
```

* `Plan_from_binding`: Whether this statement uses the bound execution plans.
* `Has_more_results`: Whether this statement has more results to be fetched by users.
* `Rewrite_time`: The time consumed for rewriting the query of this statement.
* `Preproc_subqueries`: The number of subqueries (in the statement) that are executed in advance. For example, the `where id in (select if from t)` subquery might be executed in advance.
* `Preproc_subqueries_time`: The time consumed for executing the subquery of this statement in advance.
* `Exec_retry_count`: The retry times of this statement. This field is usually for pessimistic transactions in which the statement is retried when the lock is failed.
* `Exec_retry_time`: The execution retry duration of this statement. For example, if a statement has been executed three times in total (failed for the first two times), `Exec_retry_time` means the total duration of the first two executions. The duration of the last execution is `Query_time` minus `Exec_retry_time`.
* `KV_total`: The time spent on all the RPC requests on TiKV or TiFlash by this statement.
* `PD_total`: The time spent on all the RPC requests on PD by this statement.

> * `Backoff_total`: time spent on all the backoff during the execution of this statement.

Reviewer: ```suggestion
* `Backoff_total`: The time spent on all the backoff during the execution of this statement.
```
